web/src/app/users/UserRouter.js

```diff
@@ -42,9 +42,7 @@ export default function UserRouter() {
       <Route
         exact
         path='/users/:userID'
-        render={({ match }) => (
-          <UserDetails userID={match.params.userID} readOnly />
-        )}
+        component={() => <UserDetails readOnly />}
```

Comment:
Should this stay as `render={` since it's returning the component as JSX from the function?

Reply:
Yep should be a `render`, got that switched back.